tests/checkbox/checkbox-size.html

```diff
@@ -22,133 +22,133 @@
         <span></span>
         <div>
             <span class="k-checkbox-wrap">
-                <input type="checkbox" class="k-checkbox k-checkbox-sm k-rounded-md k-checked" checked="">
+                <input id="ch1" type="checkbox" class="k-checkbox k-checkbox-sm k-rounded-md k-checked" checked="">
```

Comment:
id-s should be unique, please update the _**checkbox-size**_ test and remove the duplication